lms/templates/register-form.html

```diff
@@ -81,34 +81,34 @@ <h2 class="sr">${_('Required Information')}</h2>
+            <label for="email" class="f-16 h-26 d-flex">${_('E-mail')}<span class="text-navoica-red" aria-hidden="true">&nbsp;*</span><span class="sr-only">${_('required')}</span></label>
             <input class="form-control js-register-field" id="email" type="email" name="email" value="${email}" spellcheck="false" autocomplete="email" required aria-required="true"/>
                         <small aria-atomic="true" aria-live="assertive" class="f-14 h-19 js-error-message text-navoica-red d-flex flex-nowrap"></small>
         </div>
 
         % if third_party_auth.is_enabled() and running_pipeline:
 
         <div class="is-disabled field optional password form-group form-group-required field-password js-password-field" hidden>
-            <label for="password" class="f-16 h-26 d-flex">${_('Password')}<p class="text-navoica-red" aria-hidden="true">&nbsp;*</p><span class="sr-only">${_('required')}</span></label>
+            <label for="password" class="f-16 h-26 d-flex">${_('Password')}<span class="text-navoica-red" aria-hidden="true">&nbsp;*</span><span class="sr-only">${_('required')}</span></label>
             <input class="form-control js-register-field" id="password" type="password" name="password" value="" spellcheck="false" disabled required aria-required="true"/>
 
-            <div toggle="#password" class="far fa-eye field-icon toggle-password f-16 h-20 invisible" tabindex="0" role="button">
+            <a data-toggle="#password" class="far fa-eye field-icon toggle-password f-16 h-20 invisible" tabindex="0" role="button">
                 <span class="sr-only">${_('Click to show password')}</span>
-            </div>
+            </a>
 
             <small aria-atomic="true" aria-live="assertive" class="f-14 h-19 js-error-message text-navoica-red d-flex flex-nowrap"></small>
         </div>
 
         % else:
 
          <div class="field text form-group form-group-required password field-password js-password-field">
-            <label for="password" class="f-16 h-26 d-flex ">${_('Password')}<p class="text-navoica-red" aria-hidden="true">&nbsp;*</p><span class="sr-only">${_('required')}</span></label>
+            <label for="password" class="f-16 h-26 d-flex ">${_('Password')}<span class="text-navoica-red" aria-hidden="true">&nbsp;*</span><span class="sr-only">${_('required')}</span></label>
             <input style="display:none" type="password">
             <input class="form-control js-register-field" id="password" type="password" name="password" value="" spellcheck="false" required aria-required="true" aria-describedby="password-tip" autocomplete="off" />
 
-            <div toggle="#password" class="far fa-eye field-icon toggle-password f-16 h-20 invisible" tabindex="0" role="button">
+            <a data-toggle="#password" class="far fa-eye field-icon toggle-password f-16 h-20 invisible" tabindex="0" role="button">
```

Comment:
hej, a nie ma czystszego sposobu? Tu mamy link <a> bez hrefa, wiec trochę nie link, z nadanym tabindexem, które ma być czytane jako button. Czy to mógłby być natywny button?